src/Nancy/SafePathConfigurationExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Nancy
+{
+    using System.Collections.Generic;
+    using Nancy.Configuration;
+
+    /// <summary>
+    /// Contains <see cref="SafePathConfiguration"/> configuration extensions for <see cref="INancyEnvironment"/>.  
+    /// </summary>
+    public static class SafePathConfigurationExtensions
+    {
+        /// <summary>
+        /// Configures <see cref="SafePathConfiguration"/> 
+        /// </summary>
+        /// <param name="environment"><see cref="INancyEnvironment"/> that should be configured.</param>
+        /// <param name="paths">Safe paths.</param>
+        public static void Paths(this INancyEnvironment environment, params string[] paths)
```

Comment:
OK, we have a bad name somewhere, let's use that as a precedent for new APIs :wink: